proto/exocore/operator/v1/tx.proto

```diff
@@ -51,6 +52,71 @@ message OperatorOptedUSDValue {
   ];
 }
 
+// OperatorVotingPower represents the voting power for the specified operator address
+message OperatorVotingPower {
+  // operator_addr is the operator address,its type should be a sdk.AccAddress
+  string operator_addr = 1
+  [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // voting_power is the active voting power for the above operator address
+  string voting_power = 2
+  [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+}
+
+// VotingPowerSnapshot is used to record the historical voting power for AVSs
+message VotingPowerSnapshot {
+  // total_voting_power is the total voting power of specified AVS
+  string total_voting_power = 1
+  [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  // operator_voting_powers records the active voting power of all operators
+  // for the specified AVS
+  repeated OperatorVotingPower voting_power_set = 2;
+  // last_changed_height is used to indicate the height of most recent change when
+  // the voting_power_set is nil, which can help to fall back to the correct epoch height.
+  int64 last_changed_height = 3;
+  // epoch_identifier record the epoch info
+  string epoch_identifier = 4;
+  // epoch_number indicates which epoch this snapshot serve for
+  int64  epoch_number = 5;
+  // block_time records the block time when it is stored, and it is used for snapshot pruning
+  // The epoch_number would be sufficient for pruning if we don't support AVSs changing their
```

Comment:
CC @trestinlsd and @mikebraver - do we allow an AVS to change its epoch? If yes, it should be disallowed. What do you think?

Reply:
right, we can remove the `block_time` field if we don't allow an AVS to change its epoch. But I am not sure if it's acceptable for an AVS. I’m not sure if there are cases where AVS is misconfigured with the wrong epoch or where the current epoch frequency configuration doesn’t meet the requirements and needs to be changed.

Reply:
Actually it'll be easier for us to handle to use unified epoch and don't support AVS to change it

Reply:
I will remove the `block_time` and use the epoch number to address the pruning.